projects/frontend/docker-compose.yml

```diff
@@ -47,17 +50,3 @@ services:
       PLEK_SERVICE_STATIC_URI: assets.publishing.service.gov.uk
       VIRTUAL_HOST: frontend.dev.gov.uk
       BINDING: 0.0.0.0
-
```

Comment:
shouldn't `frontend-app-live` gain `account-api-app-live` as a `depends_on`?

Otherwise I'm not sure what happpens if you try to spin up `govuk-docker frontend-app` and it tries to bring up `account-api-app` when we really need `account-api-app-live` to get real content IDs?

Reply:
Sure, I'll add that